superset/viz.py

```diff
@@ -2375,6 +2375,35 @@ def get_data(self, df):
         return result
 
 
+class SpiderRadarViz(BaseViz):
+    """
+    A spider / radar chart.
+    """
+
+    viz_type = 'spider_radar'
+    verbose_name = _('Spider Radar Diagram')
+    credits = (
+        '<a href=https://www.visualcinnamon.com/>Nadieh Bremer, Data Viz Expert</a>')
+    is_timeseries = False
+
+    def get_data(self, df):
+        fd = self.form_data
+
+        metrics = fd.get('metrics')
+        groupby = fd.get('groupby')
+
+        axisMap = groupby[0]
+        valueMap = metrics[0].get('label')
+
+        created_data = []
```

Comment:
nit: `data = [{'axis': row[axisMap], 'value': row[valueMap]} for index, row in df.iterrows()]`

Reply:
still learning python - thanks, will make the changes and add a unit test